### PR TITLE
Tweaks to pass R CMD Check

### PR DESCRIPTION
**Pull request by njtierney**
* add badges to README and tweak a sentence
* use terra::predict not stats::predict
* Adds GH actions
  * Add R CMD Check actions
  * Add Coverage actions
  * Add relevant badges to README + add badges section
* Various tweaks to test files
  * do not namespace testthat
  * do not namespace sdmtools
  * remove `library(sdmtools)`
  * remove other unnecessary namespacing of terra if library(terra) already called
* use_package_doc() and place globalVariables() inside this to avoid CRAN NOTE
* use \dontrun for examples of africa mask
* remove extra #' for another \dontrun
* example_rast() --> example_raster()
* add ... docs and function arg for print method to it works as an S3 method
* add docs for param
* namespace stats and grDevices
* assign_nearest_land:
  * document parameters
  * update `mask` to `mask_object`
* add LICENSE file
* Tweaks to DESCRIPTION
  * MIT + file LICENSE.Md --> ... + LICENSE
  * import + namespace: grDevices, malariaAtlas, sf, stats, tidyselect
  * move to Remotes: multispeciesPP
  * use testthat version 3
  * ignore data_raw dir in R buildignore

### CODE REVIEW

File: R/assign_nearest_land.R [hunk 1]
```diff
@@ -23,20 +23,20 @@ assign_nearest_land <- function(dat_object,
   message("dat_object object must contain columns 'longitude' and 'latitude'.")
 
   # ensure that data object is spatial data frame
-  spat_dat <- st_as_sf(dat_object)
+  sf::spat_dat <- st_as_sf(dat_object)
```
Comment (njtierney) [inline R/assign_nearest_land.R:26]: I think this might be a typo?

Comment (njtierney) [inline R/assign_nearest_land.R:26]: I think you mean:

`spat_dat <- sf::st_as_sf(dat_object)`

Comment (geryan) [reply R/assign_nearest_land.R]: mmmhmmm

**Review by geryan — COMMENTED** [review]
So I've located and formatted the links, but I still get the warning running `check`:
```
❯ checking Rd cross-references ... WARNING
  Missing link or links in documentation object 'global_regions.Rd':
    ‘https://www.un.org/dgacm/en/content/regional-groups’
    ‘https://ourworldindata.org/grapher/who-regions’ ‘dataset’ ‘original
    data’
  
  See section 'Cross-references' in the 'Writing R Extensions' manual.
  ```

**Pull request merged**